server/views/pages/overview/partials/overviewTabActionPlanColumn.njk

```diff
@@ -85,7 +85,14 @@
         </table>
 
         {% if goal.note %}
-          <p class="govuk-body">Note: {{ goal.note }}</p>
+          {{ govukDetails({
+            summaryHtml: "View note<span class='govuk-visually-hidden'> for Goal " + loop.index + "</span>",
```

Comment:
Due to there possible being a number of links with the generic text "View note" we need hidden text here so that screen readers read out the Goal number too!